### DISCUSSION TITLE
Error writing coordinates to Exodus file.

### DISCUSSION THREAD

**Discussion by KangChenRui** [idaholab/moose#22727] [Q&A General] [answered]
Hello everyone!
 I first encountered the following error yesterday.
![image](https://user-images.githubusercontent.com/91399933/202328799-38e37f7a-6e33-49c7-a013-c467bbf67fdc.png)
The output module I set is shown in the figure below：
![image](https://user-images.githubusercontent.com/91399933/202328968-85100e9d-4682-4ac8-8c06-37869802f9e5.png)
Thanks for your help!

ChenRui

**Comment by GiudGiud** [comment, marked as answer]
Hello

Did anything else happened?
Did you run out of disk space?
Was the file modified during the simulation?

PS: please avoid screenshots in the future, this could all have been copy pasted as text

Guillaume

**Reply by GiudGiud** [reply]
do you consistently get this error?

**Reply by KangChenRui** [reply]
Hello!
I ran out of disk space.
Thanks a lot!
